### ISSUE TITLE
[Issue]: Metadata edits are being reverted

### ISSUE THREAD

**Issue by raqua** [jellyfin/jellyfin#8532]
### Please describe your bug

I have some kids series that are screen recorded from TV long time ago. It does get correctly identified but not fully. Problem is mostly names of episodes are episode numbers. I have edited this to be correct, but the metadata is reverted back to the detected wrong values after some time (few hours?). I have set my library to not refresh metadata (which is the default anyway) and I even tried to remove the metadata fetchers, but even this change gets reverted. It is driving me nuts to be honest.
I am new to Jellyfin, so not sure, maybe I am doing some stupid mistake, but this is not rocket science and I should be able to figure that, right? 

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Linux
- Virtualization: Docker
- Clients: Chromecast, Kodi, Android
- Browser: Firefox
- FFmpeg Version: Whatever you ship in the Docker image
- Playback Method: I guess all of the above options, but there is no problem with playback for me
- Hardware Acceleration: No
- Plugins: Kodi Queue
- Reverse Proxy: Caddy 2
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
There was no playback of any sort between my metadata edit and the reversion. The only thing I see in logs is old playback from 2 days ago. 
I am happy to provide additional info but at this point I don't see there is anything more to add.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sjorge** [comment]
I'm also experiencing this, hard refreshing after saving shows the correct meta data for me. If I then do further edits they are missing until another hard refresh.

**Comment by raqua** [comment]
Do not close.

**Comment by raqua** [comment]
Still not fixed of course, but it is last time that I comment here as this does not get any attention.

**Comment by srcrist** [comment]
I am enabled to reproduce this behavior, and I'm not really seeing a lot of reports for this either. I suspect that this is more of a troubleshooting issue than a bug. Are you using any sort of NFO files? Because those can override the metadata in the DB. Beyond that, I would head to the troubleshooting and support channels on matrix/discord, or to the forums, and see if someone can help you troubleshoot. 

I'll leave this open for now, but we'd need some reproducible steps to confirm it as a bug. As of right now, I'm not seeing the same behavior. Circle back here after trying to troubleshoot in the support channels if you still believe that it's a bug.

**Comment by NiklasBr** [comment]
I'm seeing the same issue with overwritten metadata. Is there a way to force Jellyfin to this metadata to the media files and/or NFO files perhaps? Like saying "I am happy with this state of affairs, make it permanent"?

**Comment by srcrist** [comment]
Jellyfin should do exactly this already. If you make a manual edit and lock the field, Jellyfin should not modify it on a refresh unless the media is removed and recreated entirely. Which is the behavior that I am seeing in my testing. Correctly structured NFO metadata should also be consistent. The issue here is that I'm not able to reproduce the inconsistency.

**Comment by splendid25** [comment]
Do you have NFO file in your library?
What's the modified date on the NFO files? I think when they get changed (by Radarr for example), Jellyfin picks metadata from it, even when fields are locked.

**Comment by NiklasBr** [comment]
This has not been fixed unfortunately, was still a problem last week.

**Comment by bensonchow123** [comment]
Is this fixed?

**Comment by felix920506** [comment]
This should have been fixed by #11743
